client/src/components/landing/HeroSection.tsx

```diff
@@ -19,6 +23,29 @@ const HeroSection: React.FC = () => {
           </Badge>
         </Link>
       </div>
+      <Button onClick={() => setShowLogin(true)}>Login</Button>
+      <Dialog open={showLogin} onClose={(val) => setShowLogin(val)} static={true}>
+        <DialogPanel>
+          <h2 className="text-lg font-semibold text-tremor-content-strong dark:text-dark-tremor-content-strong">Login</h2>
+          <div className="mx-auto max-w-sm space-y-8 mt-2">
+            <div>
+              <TextInput placeholder="Type your username here" error={error} errorMessage="Wrong username" />
+            </div>
+            <div>
+              <TextInput placeholder="Type password here" type="password" />
+            </div>
+          </div>
+
+          <div className="mt-8 flex items-center justify-end space-x-2">
+            <Button size="xs" variant="secondary" onClick={() => setShowLogin(false)}>
```

Comment:
I think the button's position is not good, perhaps on the top-right is better!

Reply:
Thank you for the feedback! I tried to apply your suggestion, but I noticed that it affects the symmetry of the code, which slightly impacts its aesthetics.